2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,18 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  const missingFromThePantry = {};
+  missingFromThePantry.name = recipe.name;
+  missingFromThePantry.items = [];
+    missingFromThePantry.items = recipe.ingredients.filter ((Ingredient) => {
+    if (!pantry.fridgeContents.includes(Ingredient) && !pantry.cupboardContents.includes(Ingredient)){
+      return true;
+    }
```

Comment:
instead of using the if statement we could use an or function instead within the filter function